cvmfs/swissknife_sync.cc

```diff
@@ -678,16 +679,10 @@ int swissknife::CommandSync::Main(const swissknife::ArgumentList &args) {
 
   // Start spooler
   upload::SpoolerDefinition spooler_definition(
-      params.spooler_definition,
-      hash_algorithm,
-      params.compression_alg,
-      params.generate_legacy_bulk_chunks,
-      params.use_file_chunking,
-      params.min_file_chunk_size,
-      params.avg_file_chunk_size,
-      params.max_file_chunk_size,
-      params.session_token_file,
-      params.key_file);
+      params.spooler_definition, hash_algorithm, params.compression_alg,
```

Comment:
Here, maybe the original was better, the formatter just automatically change them...